packages/visibility_detector/test/widget_test.dart

```diff
@@ -57,11 +56,27 @@ void main() {
       expect(info.visibleFraction, 1.0);
 
       final bounds =
-          VisibilityDetectorController.instance.widgetBoundsFor(cellKey);
+      VisibilityDetectorController.instance.widgetBoundsFor(cellKey);
```

Comment:
Also unintentional?

Reply:
Yes, also unintentional by commenting and un-commenting that part. I have reverted it now.